e2e/rook-test-framework/rook-infra-manager/scripts/rook-dind-cluster-v1.5.sh

```diff
@@ -0,0 +1,1042 @@
+#!/bin/bash
```

Comment:
it was for verifying that something had changed for between 1.5 and 1.6 Ill remove it...